lib/src/utils/models/encrypted_file_info.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
+import 'package:matrix/src/utils/models/encrypted_file_key.dart';
+
+part 'encrypted_file_info.g.dart';
+
+@JsonSerializable()
```

Comment:
I don't think this class needs serializable
Also no need json packages, then can keep this package lightweight

Reply:
I think it need to, you can see the request example here. 
https://spec.matrix.org/v1.7/client-server-api/#sending-encrypted-attachments

Reply:
thanks

Reply:
> https://github.com/linagora/matrix-dart-sdk/blob/main/lib/src/event.dart#L226

this is very hard to maintains and also to read :)). IMO, we should not do like them. also, i use `  json_annotation: ^4.7.0
 , json_serializable: ^6.5.4` not `json`

Reply:
got it, let's discuss in daily